jira-feature-collector/docker/jira-properties-builder.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+# mongo container provides the HOST/PORT
+# api container provided DB Name, ID & PWD
+if [ "$TEST_SCRIPT" != "" ]
+then
+	#for testing locally
+	PROP_FILE=application.properties
+else
+	PROP_FILE=hygieia-jira-feature-collector.properties
+fi
+
+if [ ! -d logs ] 
+then
+	mkdir logs
+fi
+LOG=logs/$PROP_FILE.log
+
+# if we are linked, use that info
+if [ "$MONGO_PORT" != "" ]; then
+  # Sample: MONGO_PORT=tcp://172.17.0.20:27017
+  export SPRING_DATA_MONGODB_HOST=`echo $MONGO_PORT|sed 's;.*://\([^:]*\):\(.*\);\1;'`
+  export SPRING_DATA_MONGODB_PORT=`echo $MONGO_PORT|sed 's;.*://\([^:]*\):\(.*\);\2;'`
+else
+	env >>$LOG
+	echo "ERROR: MONGO_PORT not defined" > $LOG
+	exit 1
+fi
+
+echo "SPRING_DATA_MONGODB_HOST: $SPRING_DATA_MONGODB_HOST"
+echo "SPRING_DATA_MONGODB_PORT: $SPRING_DATA_MONGODB_PORT"
+
+
+cat > $PROP_FILE <<EOF
+#Database Name
+database=${HYGIEIA_API_ENV_SPRING_DATA_MONGODB_DATABASE:-dashboard}
+
+#Database HostName - default is localhost
+dbhost=${SPRING_DATA_MONGODB_HOST:-10.0.1.1}
+
+#Database Port - default is 27017
+dbport=${SPRING_DATA_MONGODB_PORT:-9999}
+
+#Database Username - default is blank
+dbusername=${HYGIEIA_API_ENV_SPRING_DATA_MONGODB_USERNAME:-db}
+
+#Database Password - default is blank
+dbpassword=${HYGIEIA_API_ENV_SPRING_DATA_MONGODB_PASSWORD:-dbpass}
+
+#Collector schedule (required)
+feature.cron=0 * * * * *
```

Comment:
These where all in the README example, maybe the need for them can be eliminated to common set of env variables across the modules, i.e. HYGIEIA_API_ENV_SPRING_DATA_MONGODB_USERNAME, this would require separate tickets for each module I guess..  What do you think?